client/apps/edit/components/admin/index.coffee

```diff
@@ -349,13 +349,15 @@ module.exports = class EditAdmin extends Backbone.View
     @article.save()
 
   setupPublishDate: ->
-    $('.edit-admin-input-date').on 'blur', =>
-      @formatAndSetPublishDate($('.edit-admin-input-date').val())
+    $('.edit-admin-input-date, .edit-admin-input-time').on 'blur', =>
+      dateAndTime = @$('.edit-admin-input-date').val() + ' ' + @$('.edit-admin-input-time').val()
+      @formatAndSetPublishDate dateAndTime
       @article.save published_at: @saveFormat
       false
     @formatAndSetPublishDate @article.get('published_at')
 
   formatAndSetPublishDate: (date) ->
+    date = new Date(date).toISOString()
```

Comment:
Moment was throwing a deprecation warning when I ran the test initially bc I guess they're not going to be as liberal in the formatting they'll accept to convert in the next release 
![screen shot 2015-12-04 at 6 06 59 pm](https://cloud.githubusercontent.com/assets/6501236/11603686/da10ee98-9ab1-11e5-8198-03a93b550af4.png)

Adding this line resolved it